packages/flutter/lib/src/widgets/async.dart

```diff
@@ -318,12 +343,12 @@ typedef AsyncWidgetBuilder<T> = Widget Function(BuildContext context, AsyncSnaps
 /// of the following snapshots that includes the last one (the one with
 /// ConnectionState.done):
 ///
-/// * `new AsyncSnapshot<int>.withData(ConnectionState.waiting, null)`
```

Comment:
Good catch!  This must be old.

Reply:
we really should find a way to analyze this